pkg/compose/loader.go

```diff
@@ -48,7 +50,7 @@ func (s *composeService) LoadProject(ctx context.Context, options api.ProjectLoa
 		}
 	}
 
-	if options.Compatibility {
+	if options.Compatibility || utils.StringToBool(projectOptions.Environment["COMPOSE_COMPATIBILITY"]) {
```

Comment:
this should use ComposeCompatibility const but this would create an import cycle